pages/dkp/kommander/1.1/release-notes/index.md

```diff
@@ -19,6 +19,34 @@ To get started with Kommander, [download](/dkp/konvoy/latest/download/) and [ins
 
 Kommander provides a command center for all your cloud native management needs in public Information as a Service (IaaS), on-premises, and edge environments. Kommander provides a multi-tenant experience to create, secure, and configure Kubernetes clusters and cloud native workloads. Additionally, Kommander enables teams to unlock federated and cost management, across multiple clusters, whether they are a new Konvoy cluster or existing 3rd party/DIY distribution.
 
+## Improvements in Kommander 1.1.3
+
+**D2iQ&reg; Kommander&reg; version 1.1.3 was released on 25, March 2021.**
+
+# New features
+
+
+# Bug fixes
+
+- Ensure pre-delete hook jobs are cleaned up.
```

Comment:
I believe this is the fix for COPS-6821. (https://jira.d2iq.com/browse/COPS-6821)

```suggestion
- Ensure pre-delete hook jobs are cleaned up (COPS-6821).
```